packages/dashboard-core-plugins/src/panels/DropdownFilterPanel.tsx

```diff
@@ -131,7 +196,7 @@ class DropdownFilterPanel extends Component {
       prevState.valuesTable !== null
     ) {
       log.debug('Table in state modified, closing the old table.');
-      prevState.valuesTable.close();
+      prevState.valuesTable?.close();
```

Comment:
Shouldn't need this.
The outer `if` check should check if it `!== undefined`, not `!== null` (due to the changes you've made `valuesTable` can be `TableTemplate | undefined`, it can't be `null`).